cmake/BuildBenchmark.cmake

```diff
@@ -0,0 +1,60 @@
+set(BENCHMARK_VERSION v1.9.0)
+set(BENCHMARK_HASH 21a2604efeded8b4cbabc72f3e1c7a2a)
+set(BENCHMARK_INSTALL_DIR ${BUNDLED_LIBS_INSTALL_DIR}/benchmark-prefix)
+set(BENCHMARK_INCLUDE_DIR ${BENCHMARK_INSTALL_DIR}/dest/include)
+
+set(BENCHMARK_LIB ${BENCHMARK_INSTALL_DIR}/build/src/libbenchmark.a)
+set(BENCHMARK_LIB_MAIN ${BENCHMARK_INSTALL_DIR}/build/src/libbenchmark_main.a)
+
+list(APPEND BENCHMARK_CMAKE_FLAGS
+     "-DBENCHMARK_ENABLE_TESTING=OFF"
+     "-DBENCHMARK_ENABLE_LTO=OFF"
+     "-DBENCHMARK_USE_LIBCXX=OFF"
+     "-DBENCHMARK_ENABLE_GTEST_TESTS=OFF"
+     "-DBENCHMARK_ENABLE_LIBPFM=OFF"
+     # By default, this library is built in Debug. Use the
+     # non-Debug build for benchmarks.
```

Comment:
Also, building benchmarks in Debug is forbidden in a root CMakeLists.txt:
```
if (CMAKE_BUILD_TYPE STREQUAL "Debug")
    message(AUTHOR_WARNING "Benchmarks are available only in release build")
```

Reply:
I've updated the comment to avoid confusion.
```diff
diff --git a/cmake/BuildBenchmark.cmake b/cmake/BuildBenchmark.cmake
index b5a7541458..d68441c89c 100644
--- a/cmake/BuildBenchmark.cmake
+++ b/cmake/BuildBenchmark.cmake
@@ -12,8 +12,9 @@ list(APPEND BENCHMARK_CMAKE_FLAGS
      "-DBENCHMARK_USE_LIBCXX=OFF"
      "-DBENCHMARK_ENABLE_GTEST_TESTS=OFF"
      "-DBENCHMARK_ENABLE_LIBPFM=OFF"
-     # By default, this library is built in Debug. Use the
-     # non-Debug build for benchmarks.
+     # By default, this library is built in Debug. Propagate the
+     # non-Debug build for benchmarks (benchmarks aren't built in
+     # the Debug mode).
      "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
 )
 
```

Reply:
Thanks!